tools/import_validation/runner.py

```diff
@@ -72,53 +77,66 @@ def __init__(self, validation_config_path: str, differ_output: str,
                 (self.validator.validate_max_value_check, 'stats'),
         }
 
-    def _initialize_data_sources(self, stats_summary: str, differ_output: str):
+        self._initialize_data_sources(stats_summary, lint_report, differ_output)
+
+    def _initialize_data_sources(self, stats_summary: str, lint_report: str,
+                                 differ_output: str):
         """
     Checks for and loads the required data sources based on the config.
     """
-        stats_required, differ_required = self._determine_required_sources()
+        req_sources = self._determine_required_sources()
 
-        if stats_required and (not stats_summary or
-                               not os.path.exists(stats_summary)):
+        if 'stats' in req_sources and (not stats_summary or
+                                       not os.path.exists(stats_summary)):
             raise ValueError(
                 f"A validation rule requires the 'stats' data source, but the --stats_summary file was not provided or does not exist. Path: {stats_summary}"
             )
 
-        if differ_required:
+        if 'differ' in req_sources:
             if not differ_output or not os.path.exists(differ_output):
                 logging.warning(
                     f"A validation rule requires the 'differ' data source, but the --differ_output file was not provided or does not exist. Path: {differ_output}"
                 )
 
+        if 'lint' in req_sources and (not lint_report or
+                                      not os.path.exists(lint_report)):
+            raise ValueError(
+                f"A validation rule requires the 'lint' data source, but the --lint_report file was not provided or does not exist. Path: {lint_report}"
+            )
+
         if stats_summary and os.path.exists(stats_summary) and os.path.getsize(
                 stats_summary) > 0:
-            self.dataframes['stats'] = pd.read_csv(stats_summary)
+            self.data_sources['stats'] = pd.read_csv(stats_summary)
         elif stats_summary and os.path.exists(stats_summary):
             logging.warning("stats_summary file exists but is empty: %s",
                             stats_summary)
 
         if differ_output and os.path.exists(differ_output) and os.path.getsize(
                 differ_output) > 0:
-            self.dataframes['differ'] = pd.read_csv(differ_output)
+            self.data_sources['differ'] = pd.read_csv(differ_output)
         elif differ_output and os.path.exists(differ_output):
             logging.warning("differ_output file exists but is empty: %s",
                             differ_output)
 
-    def _determine_required_sources(self) -> Tuple[bool, bool]:
+        if lint_report and os.path.exists(lint_report) and os.path.getsize(
+                lint_report) > 0:
+            with open(lint_report, 'r') as f:
+                self.data_sources['lint'] = json.load(f)
```

Comment:
can we catch any json parse exceptions and log an error with the full file path?